packages/api-explorer/src/block-types/Content.jsx

```diff
@@ -128,6 +128,7 @@ Loop.defaultProps = {
 
 Content.propTypes = {
   body: PropTypes.string,
+  copyButtons: PropTypes.bool,
```

Comment:
nit: can we rename to `isCopyEnabled` or something?

Reply:
Yea, but to be consistent, I'll have to change it in markdown first, then api-explorer and readme.

Reply:
ok up to you!